frontend-public/src/components/dashboard/mine/variances/Variances.js

```diff
@@ -49,6 +68,80 @@ export class Variances extends Component {
     this.props.fetchVarianceStatusOptions();
   }
 
+  handlAddDocuments = (files, varianceGuid) => {
+    Promise.all(
+      Object.entries(files).map(([document_manager_guid, document_name]) =>
+        this.props.addDocumentToVariance(this.props.mine.mine_guid, varianceGuid, {
+          document_manager_guid,
+          document_name,
+        })
+      )
+    );
+  };
+
+  handleAddVariances = (files) => (values) => {
+    const received_date = values.received_date
+      ? values.received_date
+      : moment().format("YYYY-MM-DD");
+    const newValues = { received_date, ...values };
```

Comment:
If `received_date` exists on `values` then I think this is just going to overwrite `received_date` with `values.received_date`, so `newValues` will be equal to `values` in that case

Reply:
Ah good catch, that should actually just be set to the current date, received_date will never be filled in by the user. 

updated it to be: 
```
const received_date = moment().format("YYYY-MM-DD");
```